lib/mercadopago/api/clients.rb

```diff
@@ -3,45 +3,46 @@
 require '../base'
 
 module MercadoPago
-  class Clients < Base
-
-    # Override parent class method
-    # Place here main base URL
-    def service_url
-      'https://api.mercadopago.com'
-    end
+  module API
+    class Clients < Base
```

Comment:
MercadoPago::API::Clients assumes too much for instance variable '@access_token'